src/main/java/net/mcreator/io/zip/ZipIO.java

```diff
@@ -36,13 +36,11 @@
 
 	private static final Logger LOG = LogManager.getLogger("ZipIO");
 
-	public static void iterateZip(File zipFilePointer, IZipFileAction action) {

```

Comment:
This is the only usage of `IZipFileAction`, the reason I removed it.

Reply:
Reasonable :)